AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModule.swift

```diff
@@ -15,7 +15,7 @@ let incomingCallKitDelayInSeconds = 10.0
 class MeetingModule {
     private static var sharedInstance: MeetingModule?
     private(set) var activeMeeting: MeetingModel?
-    private let meetingPresenter = MeetingPresenter()
+    private let meetingPresenter = MeetingPresenter.shared()
```

Comment:
MeetingPresenter should not be used outside the scope of MeetingModule.
Can we present the debug setting view directly from joinViewController?
If we want to get fancy, we can create a Joining Module and presenter, but I don't think it's necessary at this point